gazelle/python/parser.go

```diff
@@ -38,20 +38,7 @@ var (
 )
 
 func startParserProcess(ctx context.Context) {
-	rfiles, err := runfiles.New()
-	if err != nil {
-		log.Printf("failed to create a runfiles object: %v\n", err)
-		os.Exit(1)
-	}
-
-	parseScriptRunfile, err := rfiles.Rlocation("rules_python_gazelle_plugin/python/parse")
-	if err != nil {
-		log.Printf("failed to initialize parser: %v\n", err)
-		os.Exit(1)
-	}
-
-	cmd := exec.CommandContext(ctx, parseScriptRunfile)
-	cmd.Env = append(os.Environ(), rfiles.Env()...)
+	cmd := exec.CommandContext(ctx, "python3", pyzPath, "parse")
```

Comment:
nit: it may be good to add a comment stating why this is system python and mention #691 as the ticket which may change this need.